docs/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,33 @@
+{#
```

Comment:
This is copied from Qiskit SDK, other than where I comment. I'm probably going to update Qiskit to mirror this